iphone/Classes/UtilsModule.m

```diff
@@ -90,20 +90,32 @@ - (NSString *)md5HexDigest:(id)args
 - (id)sha1:(id)args
 {
   ENSURE_SINGLE_ARG(args, NSObject);
-  NSString *nstr = [self convertToString:args];
-  const char *cStr = [nstr UTF8String];
+
+  NSData *data;
+  if ([args isKindOfClass:[TiBlob class]]) {
+    data = [(TiBlob *)args data];
+  } else {
+    data = [[self convertToString:args] dataUsingEncoding:NSUTF8StringEncoding];
+  }
```

Comment:
Move to `convertToData:` ?
```objc
- (NSData *)convertToData:(id)arg
{
  if ([args isKindOfClass:[TiBlob class]]) {
    return [(TiBlob *)args data];
  } else {
    return [[self convertToString:args] dataUsingEncoding:NSUTF8StringEncoding];
  }
}
```
And maybe even handle Ti.File objects here as well, similar to what we do in `convertToString:`.